src/ontology/mondo-edit.obo

```diff
@@ -71,6 +71,14 @@ property_value: IAO:0000700 MONDO:0021125
 property_value: IAO:0000700 MONDO:0021178
 property_value: IAO:0000700 MONDO:0042489
 
+[Term]
+id: MONDO:0000000
+name: TRIP11 related skeletal dysplasia
```

Comment:
for all of the gene related diseases, we usually write them as TRIP11-related skeletal dysplasia (with a dash), could you please revise this as such and add this label as an exact synonym? Thanks!